ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -389,6 +389,8 @@ internal class PaywallViewModelImpl(
                 variableDataProvider = variableDataProvider,
                 activelySubscribedProductIdentifiers = customerInfo.activeSubscriptions,
                 nonSubscriptionProductIdentifiers = customerInfo.nonSubscriptionTransactions
+                    // Filter out consumables so they don't get checked in valiation
+                    .filter { !it.shouldConsume }
```

Comment:
Hmm I'm honestly wondering if we should just remove the validation for repurchasing any product and just leave it to google to display a "Already purchased product" if the customer is already subscribed/has a non-consumed product...

Reply:
+1

Reply:
Did we have this conversation already on my prototype of this from months ago? Because this feels awkwardly familiar? 🤔